packages/client/src/runtime/core/init/checkPlatformCaching.ts

```diff
@@ -0,0 +1,39 @@
+import Debug from '@prisma/debug'
+import { PrismaClientInitializationError } from '@prisma/engine-core'
+
+import type { GetPrismaClientConfig } from '../../getPrismaClient'
+
+const debug = Debug('prisma:client')
+
+/**
+ * Known platforms that have caching issues. Updating this list will also update
+ * the error message and the link to the docs, so add docs/links as needed.
+ */
+const cachingPlatforms = {
+  Vercel: 'vercel',
+  'Netlify CI': 'netlify',
+} as const
```

Comment:
I think they key here is the `ci-info` identifier of that platform? 
If so, we should document that so it is clear how this can be expanded if needed.